docs/UserGuide.md

```diff
@@ -288,8 +291,10 @@ The `required` command allows you to check the modules of the Computer Science c
 To check required modules:
 
 1. Type `required` into the command box, press **Enter** to execute it.
+![Required1](images/Required1.png)
 
-2. The result display box will then display all the modules you have yet to take for each particular field, as shown below.
```

Comment:
I think we can leave the “as shown below” inside, as it helps improve navigability.

Reply:
ahhh okok i just thought it might be a little repetitive but ok i'll put it back hehe